CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## Unreleased
 
+- Methods for calling the agent-state APIs"
```

Comment:
random quote at the end of this line

Reply:
I've updated this in ebd235272b9b85156655f837475a2ec872406253